basxconnect/mailer_integration/migrations/0020_subscription_deleted.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.4 on 2022-01-05 07:01
```

Comment:
Normally I try to merge multiple migrations, if possible. It can easily be done with:

```bash
./manage.py migrate <appname> XXXX
# delete all uncommited migration files
./manage.py makemigrations <appname>
./manage.py migrate <appname>
```
Where XXXX stands for the last migration number that is on the main branch. That way you will get a single, combined migration file.
But no change request, just as a future hint, and it can help to reduce the number of migration files a bit.

Reply:
I agree